### PR TITLE
Fix russian translation

### PR DESCRIPTION
**Pull request by swarnk**

### CODE REVIEW

**Comment by 5bentz** [comment]
Thanks for the translation!
I cannot verify it though (I don't speak Russian).

However, I see that the Russian translation is missing 15 entries.
I have done a pull request with the missing entries on your branch.
Could you also fix them? Thank you in advance :+1:

**Comment by swarnk** [comment]
I made several changes. Have I done everything right?

**Comment by swarnk** [comment]
I'm not entirely sure whether I did everything correctly

File: src/_locales/ru/messages.json [hunk 1]
```diff
@@ -165,12 +195,18 @@
...
   "passff_newpassword_errors_unexpected_error": {
     "message": "Извините, произошла неожиданная ошибка"
   },
-  "passff_errors_unexpected_error": {
-    "message": "Извините, произошла неожиданная ошибка. Убедитесь что ваши установки правильны и что нативная программа заинстлирована в операционной системе."
+  "passff_errors_exec_host_app": {
+    "message": "Соединение с хост-приложением не удалось или возвратило неожиданный результат!"
+  },
+  "passff_errors_upgrade_host_app": {
+    "message": "Убедитесь, что вы используете последнюю версию хост-приложения PastFF. Смотри руководство по установке {installation instructions on GitHub}."
```
Comment (5bentz) [inline src/_locales/ru/messages.json]: ^ These aren't the brackets you are looking for.

**Comment by tuxor1337** [comment]
> As far as I know, the rule of thumb is: "translators mustn't translate strings in brackets cause their are variables". While in PassFF, you have to translate it.

That's interesting. How would you usually do that kind of thing that we are trying to achieve with braces?

**Comment by 5bentz** [comment]
I tested the translation. It works.

@tuxor1337 Sorry I wasn't clear. I was talking about translation rules on [Launchpad](https://help.launchpad.net/Translations/Guide) for example.
In some translations, you may find HTML code for links: `<a href=url_not-translated>String to translate</a>`

**Comment by tuxor1337** [comment]
@5bentz Okay, allowing for HTML in translations would mean that we need an additional layer of sanitization (because of deliberate or accidental code injections by translators). But probably that's the way to go long term. However, now that we are still a small project with few translations it doesn't really matter. Furthermore, only one single string is affected by this.

**Comment by swarnk** [comment]
Translated thid bit.
https://github.com/swarnk/passff/commit/a795ba515262a1f376a796741ff1e90e59f9a677

**Comment by 5bentz** [comment]
https://github.com/5bentz/passff/tree/fix-russian2

**Comment by 5bentz** [comment]
https://github.com/5bentz/passff/commit/4276843fe6b6d2ca65a4db727adf176c835c7e64

**Comment by 5bentz** [comment]
Thanks!
Meanwhile, two new translations have been added.

To update the translation file, you can rebase this branch. Assuming this branch is your branch `master`, you could do:
```
git checkout master
# if not done yet, add the official repository
git remote add official https://github.com/passff/passff.git
git fetch official
git rebase official/master
```

**Comment by tuxor1337** [comment]
@swarnk Thanks a lot for your translation also from my side! It would be great if you could add the two new translatable strings that have been added in the meantime.

**Comment by 5bentz** [comment]
@tuxor1337 
(This is an answer to your message in #337)
2 untranslated strings is better than 17. Therefore, I'd suggest to merge it anyway, right before the release.

I managed to add the missing entries and push to swarnk's branch!
@swarnk You just have to translate the last 2 strings (with the GitHub interface for example!). Thanks for the good work!

**Pull request merged**